### PR TITLE
uiwebview problem

### PR DESCRIPTION
**Pull request by BBEDERRAR**
## Description

Fixed issue #<issue-number>

Added `yourNewMethodName()` that allows ...

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |
| Windows |    ✅❌     |

## Checklist

* [ ] I have tested this on a device/simulator for each compatible OS
* [ ] I added the documentation in `README.md`
* [ ] I mentioned this change in `CHANGELOG.md`
* [ ] I updated the typings files (`deviceinfo.d.ts`, `deviceinfo.js.flow`)
* [ ] I updated the dummy web/test polyfill (`default/index.js`)
* [ ] I added a sample use of the API (`example/App.js`)

### CODE REVIEW

File: ios/RNDeviceInfo/RNDeviceInfo.m [hunk 1]
```diff
@@ -398,6 +401,33 @@ - (NSDictionary *)powerState
     resolve(@(locationServicesEnabled));
 }
 
+RCT_EXPORT_METHOD(getUserAgent:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
```
Comment (gundlev) [inline ios/RNDeviceInfo/RNDeviceInfo.m:404]: where did you get this function from and what do you use it for?

**Comment by ashtrus** [comment]
could you add a link to merge that fixes the issue.

**Comment by BBEDERRAR** [comment]
https://github.com/react-native-community/react-native-device-info/pull/757/files

File: ios/RNDeviceInfo/RNDeviceInfo.m (hunk 1 above)
Comment (BBEDERRAR) [reply ios/RNDeviceInfo/RNDeviceInfo.m:404]: I got it from a PR from the official package
https://github.com/react-native-community/react-native-device-info/pull/757/files

**Comment by gundlev** [comment]
Where did you remove the old UIwebview?

**Comment by BBEDERRAR** [comment]
@gundlev  I didn't 
I followed the same step in the PR

**Comment by gundlev** [comment]
Okay and how did you confirm that it works?

**Comment by BBEDERRAR** [comment]
I test it in the real device from local after did :
1- changed in node modules 
2- run "react-native unlink react-native-device-info " 
3- then "react-nativenlink react-native-device-info"

**Comment by gundlev** [comment]
okay, so did you test that the module still works as expected? maybe try checking where we use it and see if it still gives the expected results. And also test if this works on all supported iOS devices :)

**Comment by BBEDERRAR** [comment]
I test it in the test device and emulator 
I think this package is used everywhere in the app because we are using it the file styles.js (app/commun)

But as you said we need to test it in the other version xD

**Comment by gundlev** [comment]
Screenshot 2020-02-11 at 16 15 22
its just used here, just test there :)

**Comment by BBEDERRAR** [comment]
Yes I mean that the package exists in styles.js (app/commun) and that file is a principal file in the app

**Comment by gundlev** [comment]
Yes, but if you just check that these functions behave as they should, you should be fine in regards to styles.js
Screenshot 2020-02-11 at 16 18 41

**Comment by BBEDERRAR** [comment]
Okay

**Comment by BBEDERRAR** [comment]
@gundlev  I have tested the app in the iPhone X and IPad and it works fine

**Pull request merged**